components/rhs_comment/rhs_comment.jsx

```diff
@@ -158,17 +158,16 @@ export default class RhsComment extends React.PureComponent {
         let profilePicture;
         let visibleMessage;
 
-        let userProfile = null;
-        if (!isConsecutivePost) {
```

Comment:
Why not keep it as before and add here something like:

```js
let userProfile = null;
if (this.props.compactDisplay) {
            userProfile = (
                <UserProfile
                    userId={post.user_id}
                    isBusy={this.props.isBusy}
                    isRHS={true}
                    hasMention={true}
                />
            );
}
```
and then, the previous approach with:

```js
if (!isConsecutivePost) {
            userProfile = (
                <UserProfile
                    userId={post.user_id}
                    isBusy={this.props.isBusy}
                    isRHS={true}
                    hasMention={true}
                />
            );
            ...
```

Reply:
I confirmed that is not a big deal because the Popover is render on visualization. Anyway, feel free to add my proposed optimization, when you multiply that component for 100 for example in a long thread, probably it has some impact.